### ISSUE TITLE
Random heavy class Zirax ship spawning as you approach a Pentaxid asteroid in space

### ISSUE THREAD

**Issue by byo13** [RavienGaming/Reforged-Eden-2#380]
### Reforged Eden 2 Build Number

23

### Game mode

Single Player

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

Sometimes, when approaching a Pentaxid Asteroid in space, a heavy class ship will spawn close to the asteroid, right in front of you, guarding the same asteroid, along with some usual scout ships. This leaves you no chance to get away safely, especially when you already parked for a mining operation.

### POI Name

Pentaxid Asteroid

### Playfield Name

Any

### Playfield Class

Any

### Game Seed

249663

**Comment by RavienGaming** [comment]
The only way a OPV can spawn is via the PDA or space defense system. Are there any Zirax POIs near that asteroid that could call a space defense?

**Comment by byo13** [comment]
The weird thing is that there wasn't any POIs around as I was looking for a safe spot for mining. It seems that it was the same behaviour as those small Infiltrator ships that usually spawn near asteroids and I thought the asteroid itself was a trap (unless it is, of course). There was nothing on my 12000m scanner. And then I just saw the ship practically glued to the asteroid, maybe stuck around it. It happened only once but I lost my ship and died as I was EVA. I will pay attention if it happens again.

**Comment by RavienGaming** [comment]
Btw drones don't spawn around asteroids either. Nothing spawns in a playfield except at initial playfield generation, or via the PDA or space reinforcement systems. Nothing respawns either.

**Comment by RavienGaming** [comment]
Most likely the ship had patrolled to the asteroid and decided to park on it for... Reasons.

Either way not anything we can fix within the scenario. Ship AI is 100% hard coded, except for a few parameters that can be set in the playfield files. Collision avoidance is not one of them unfortunately.
